Forms/MainForm.cs

```diff
@@ -1063,6 +1231,23 @@ private void BLoadPatch_Click(object sender, EventArgs e)
             tPatch.Text = _settings.InputPatchFilename;
         }
 
+        private void bLoadPreset_Click(object sender, EventArgs e)
```

Comment:
Can we change these from UI buttons to menu items instead, so we don't clutter up the interface?